core/templates/domain/objects/FractionObjectFactorySpec.ts

```diff
@@ -42,6 +42,18 @@ describe('Fraction Object Factory', () => {
     expect(createdFraction.toDict()).toEqual(fractionObject);
   });
 
+  it('should equate two fractions', () => {
```

Comment:
What about tests for unequal case?

Reply:
Added! For some reason, I forgot to push those tests. Thanks for catching this.